features/full_tests/batch_1/detect_anr_jvm.feature

```diff
@@ -14,25 +14,6 @@ Scenario: ANR triggered in JVM loop code is captured
     And the error payload field "events.0.threads.0.type" equals "android"
     And the error payload field "events.0.threads.0.stacktrace.0.type" is null
 
-# Other scenarios use a deadlock to generate an ANR, which works on Samsung devices. This scenario remains skipped
-# on Samsung as it is explicitly design to test ANRs caused by a sleeping thread.
-@skip_samsung
-Scenario: ANR triggered in JVM sleep code is captured
-    When I run "JvmAnrSleepScenario"
```

Comment:
There still need to be two ANR scenarios if it's at all possible IMO, giving coverage of:

1. An ANR triggered where the method is native (e.g. Thread.sleep)
2. An ANR triggered where the method originates in the JVM

Would it be possible to trigger an ANR by something like `while (true) {}` to maintain this coverage? If not I have a couple of ideas such as using a `BroadcastReceiver` to trigger an ANR.